api/src/wfl/service/cromwell.clj

```diff
@@ -281,8 +277,13 @@
   Return:
    List of UUIDS for each workflow as reported by cromwell."
   [environment wdl imports-zip inputs options labels]
-  ; todo: call the batch endpoint
-  (mapv #(submit-workflow environment wdl imports-zip % options labels) inputs))
+  (->> (partify-workflow wdl
+                         imports-zip
+                         inputs
+                         options
+                         labels)
+       (post-workflow (str (api environment) "/batch"))
+       (mapv #(:id %))))
```

Comment:
Just want to check that that the order of the UUIDs returned by `submit-workflows` matches the order of workflows passed in?

Reply:
That's what I noticed from testing, but I couldn't find anything about that specifically in the Cromwell docs. I'll double check with them in slack

Reply:
So that is the behavior that the Cromwell team expects